bgpd/bgp_vty.c

```diff
@@ -7968,6 +7968,41 @@ DEFUN (no_neighbor_ttl_security,
 	return bgp_vty_return(vty, peer_ttl_security_hops_unset(peer));
 }
 
+/* disable-addpath-rx */
+DEFUN(neighbor_disable_addpath_rx, neighbor_disable_addpath_rx_cmd,
```

Comment:
the line-wrapping for the new DEFUNs is different from the one right above, for example - has our standard for these macros changed?

Reply:
I can change that according to others. This is what I got after clang-formater :)